pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -438,11 +450,37 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
         }
     }
 
+    protected void checkRateLimit(BaseCommand cmd) {
+        if (cmd.getType() == BaseCommand.Type.PONG && cmd.getType() == BaseCommand.Type.PING) {
+            return;
+        }
+        if (requestRateLimiter.acquire(1) == 0 && !pausedDueToRateLimitation) {
```

Comment:
It doesn't look like there's a way to disable this completely. Please ensure that this is a no-op when this feature is disabled.

Reply:
Setting with a negative value means disabling, I have improved the doc.

Reply:
I added another comment earlier in this method where the method would return if `rateLimitingSecondsAfterResumeFromUnreadable <= 0`